tests/high-level-tests.lisp

```diff
@@ -152,9 +152,9 @@
                                   :type '(complex double-float)))
         (tol (* 1.0d2 double-float-epsilon)))
     (multiple-value-bind (u1 u2 v1h v2h theta)
-        (magicl::csd-2x2-basic x 1 1)
+        (magicl-lapack::csd-2x2-basic x 1 1)
```

Comment:
Is this renaming going to require changes to quilc/qvm?

Reply:
`csd-2x2-basic` is only used internally in MAGICL, not in QUILC/QVM, fortunately.